addons/website/views/snippets/s_comparisons.xml

```diff
@@ -9,8 +9,8 @@
                 <p class="lead">Listing your product pricing helps potential customers quickly determine if it fits their budget and needs.</p>
             </div>
             <div class="row gap-4 gap-lg-0">
-                <div class="col-lg-4 s_col_no_bgcolor" data-name="Plan">
-                    <div class="card h-100" data-name="Card">
+                <div class="col-lg-4 s_col_no_bgcolor o_parent_handler" data-name="Plan">
+                    <div class="s_card card o_cc o_cc1 h-100 mt0" data-vxml="001">
```

Comment:
Forgot a comment I had to say: 
Because of the card footer, the card does not look good if we place the card image horizontally :grimacing:
![image](https://github.com/user-attachments/assets/e678a1f1-f103-41b6-b288-ef9448f4c5af)

It would be cool to make a rule or change the template so the footer stays below everything (or at least below the body) :slightly_smiling_face:

Reply:
Let's forget this for now, if the user doesn't like it, he can go back to image on top (there surely will be a footer option one day) :+1: